Binary files libiscsi-1.18.0/.git/index and new/.git/index differ
diff -rup libiscsi-1.18.0/lib/iser.c new/lib/iser.c
--- libiscsi-1.18.0/lib/iser.c	2018-04-26 09:51:53.869133618 -0400
+++ new/lib/iser.c	2018-04-26 10:03:45.163899456 -0400
@@ -32,6 +32,20 @@
 #include <semaphore.h>
 #include <poll.h>
 
+
+#ifndef container_of
+/**
+  * container_of - cast a member of a structure out to the containing structure
+  * @ptr:        the pointer to the member.
+  * @type:       the type of the container struct this is embedded in.
+  * @member:     the name of the member within the struct.
+  *
+ */
+#define container_of(ptr, type, member) \
+    ((type *) ((uint8_t *)(ptr) - offsetof(type, member)))
+#endif
+
+
 #ifdef __linux
 
 static int cq_handle(struct iser_conn *iser_conn);
